packages/test-cases/cases/apis/wasm-rs/asyncify/query/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+pub mod w3;
+use w3::imported::ethereum_query;
+use w3::query;
+pub use w3::*;
+
+pub fn get_data(input: query::InputGetData) -> u32 {
+    match ethereum_query::EthereumQuery::call_contract_view(
+        &ethereum_query::InputCallContractView {
+            address: input.address,
+            method: "function get() view returns (uint256)".to_string(),
+            args: None,
+            connection: input.connection,
+        },
+    ) {
+        Ok(v) => v.parse::<u32>().unwrap(),
```

Comment:
Shouldn't this be bigInt? The return is unsigned 256 bit integer and we're using a 32 bit one

Reply:
technically yes it should, but this is just for testing so it doesn't matter much, we can assume the value in the SimpleStorage contract is `< u32`.